torch/optim/lr_scheduler.py

```diff
@@ -85,21 +85,21 @@ def wrapper(*args, **kwargs):
         self._initial_step()
 
     def _initial_step(self):
-        """Initialize step counts and performs a step"""
+        """Initialize step counts and performs a step."""
         self.optimizer._step_count = 0
         self._step_count = 0
         self.step()
 
     def state_dict(self):
-        """Returns the state of the scheduler as a :class:`dict`.
+        """Return the state of the scheduler as a :class:`dict`.
 
         It contains an entry for every variable in self.__dict__ which
         is not the optimizer.
         """
         return {key: value for key, value in self.__dict__.items() if key != 'optimizer'}
 
     def load_state_dict(self, state_dict):
-        """Loads the schedulers state.
+        """Load the schedulers state.
```

Comment:
```suggestion
        """Load the scheduler's state.
```
Since we're at it, might as well fix these